src/terms/local.jl

```diff
@@ -104,26 +117,30 @@ function compute_local_potential(basis::PlaneWaveBasis{T}; positions=basis.model
     # and a structure factor e^{-i G·r}
     model = basis.model
     Gqs_cart = [model.recip_lattice * (G + q) for G in to_cpu(G_vectors(basis))]
-    # TODO Bring Gqs_cart on the CPU for compatibility with the pseudopotentials which
-    #      are not isbits ... might be able to solve this by restructuring the loop
 
-    form_factors = atomic_local_form_factors(basis, Gqs_cart)
+    form_factors, iG2norm = atomic_local_form_factors(basis, Gqs_cart)
```

Comment:
Not sure if iG2norm is the best name. It essentially maps from iG to the location where the appropriate form factor element is stored.